Sources/InfomaniakLogin/Model/ApiToken.swift

```diff
@@ -75,7 +80,8 @@ public extension ApiToken {
     }
 
     var truncatedRefreshToken: String {
-        truncateToken(refreshToken)
+        guard let refreshToken else { return "" }
```

Comment:
Maybe we can merge `truncateToken` into `truncatedRefreshToken`.
And change the guard for a : 

```swift
guard let refreshToken, 
          refreshToken.length > 4 else { 
    return "" 
}
```

prefix() will work even if the string is shorter than 4, but the result will be 'garbage'

Reply:
This was brought back from Core, I don't want to touch it in this PR. `truncateToken` is used both for the refresh token and the access token. It only has logging purposes.